PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-aad/src/main/java/com/microsoft/azure/toolkit/intellij/connector/aad/ApplicationTemplateType.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.azure.toolkit.intellij.connector.aad;
+
+import com.intellij.util.PathUtil;
```

Comment:
Can you use non-intellij API in non-UI classes when possible? Because we hope to reuse these classes as many as possible in eclipse plugin.